common/src/main/java/org/opensearch/ml/common/transport/update/MLUpdateModelCacheAction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.update;
+
+import org.opensearch.action.ActionType;
+
+public class MLUpdateModelCacheAction extends ActionType<MLUpdateModelCacheNodesResponse> {
```

Comment:
Will this action only update model in cache? Add some comment?

Reply:
Got the answer by reading more code. This action just update model in cache. 
Suggest add some comments to explain , so others can understand quickly.